docs/cedarling/cedarling-properties.md

```diff
@@ -23,7 +23,6 @@ These Bootstrap Properties control default application level behavior.
 * **`CEDARLING_MAPPING_ID_TOKEN`** : Name of Cedar id_token schema entity
```

Comment:
Can we explain at least one (or more) mapping property with an example. May be we can include here:  https://docs.jans.io/head/cedarling/cedarling-properties/#loading-from-json

Reply:
What do you think about this commit
https://github.com/JanssenProject/jans/pull/10377/commits/29c4dc46f47a9ae8557448376a0a6074b905473d